packages/dev-console-app/vite.config.ts

```diff
@@ -6,11 +6,14 @@ import reactRefresh from '@vitejs/plugin-react-refresh';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [reactRefresh()],
+  build: {
+    outDir: '../../api/dev-console-app',
+  },
   resolve: {
     alias: {
       // vite.config.ts, tsconfig.json, and jest.config.ts all need to define their own aliases
       '@': path.resolve(__dirname, './src'),
-      '@shopify/ui-extensions-dev-console': path.resolve(__dirname, '../dev-console/src'),
+      '@shopify/ui-extensions-dev-server-kit': path.resolve(__dirname, '../dev-server-kit/src'),
```

Comment:
feels confusing to have the folder name different from the package name. Could we standardize?